e2e/src/pages/HomePage.ts

```diff
@@ -0,0 +1,23 @@
+import { Page, ElementHandle } from '@playwright/test';
+import BasePage from '../base/BasePage';
+import PostComponent from '../components/PostComponent';
+
+export default class HomePage extends BasePage {
+    public postComponent: PostComponent;
+
+    constructor(page: Page) {
+        super(page, 'Home Page', '');
+        this.postComponent = new PostComponent(page);
+    }
+
+    public async getAllPreviewPosts(): Promise<ElementHandle[]> {
+        return await this.page.$$('article.entry');
+    }
+
+    public async verifyAllPreviewsVisible(): Promise<void> {
+        const previewPosts = await this.getAllPreviewPosts();
+        for (const post of previewPosts) {
+            await post.isVisible();
+        }
+    }
```

Comment:
This code looks strange. It doesn't seem to check anything.
<img width="580" alt="image" src="https://github.com/user-attachments/assets/d4c54de2-bef5-4041-ada2-f34e38af8faa" />